austation/code/modules/research/techweb/all_nodes.dm

```diff
@@ -3,6 +3,11 @@
 /datum/techweb_node/unregulated_bluespace
 	design_ids = list("desynchronizer", "bluespace_jar")
 
+
+//Cyborg upgrades
+
+/datum/techweb_node/cyborg_upg_util
+	design_ids = list("borg_upgrade_holding", "borg_upgrade_lavaproof", "borg_upgrade_thrusters", "borg_upgrade_selfrepair", "borg_upgrade_expand", "borg_upgrade_rped", "borg_upgrade_circuitapp", "borg_upgrade_cooking", "borg_upgrade_cutter")
```

Comment:
don't override entire nodes, just add it as a comment.
I'm going to make a modularized way to add shit soon.

Reply:
aight so you want me to move it to base code?